auth-foundation/src/main/java/com/okta/authfoundation/credential/Token.kt

```diff
@@ -66,7 +69,34 @@ class Token constructor(
      * The configuration that was used to fetch this token.
      */
     val oidcConfiguration: OidcConfiguration,
+    @EncodeDefault
+    internal val issuedAt: Long = oidcConfiguration.clock.currentTimeEpochSecond() - expiresIn,
```

Comment:
And to clarify, this value is only used when `issuedAt` is not supplied to the constructor?  Or is this only used in serialization decoding?

Reply:
This value is only used in serialization decoding. The other constructor is the one that's publicly supplied to the rest of the SDK